src/Runner.jl

```diff
@@ -846,7 +828,9 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
     merge!(mapping, Dict(
         "PATH" => join((
             # First things first, our compiler wrappers trump all
-            "/opt/bin",
+            "/opt/bin/$(triplet(platform))",
+            # Also wrappers for the host
+            "/opt/bin/$(triplet(host_platform))",
```

Comment:
I think you wanted to have also host wrappers in the PATH, right?  but maybe after all target wrappers?